# Corona_Virus_Genome
This project involved the analysis of seven virus genomes, including the SARS-CoV-2 genome and six other similar genomes. The genomes were provided in FASTA format (.fna files). The goal of the project was to perform various analyses on these genomes to uncover interesting findings.

Data The data for this project consisted of seven virus genomes in FASTA format. The genomes were:

* AY274119.3 Severe acute respiratory syndrome-related coronavirus isolate Tor2, complete genome
* AY278488.2 SARS coronavirus BJ01, complete genome
* KP876546.1 Rhinolophus bat coronavirus BtCoV/4991 RNA-dependent RNA polymerase (RdRp) gene, partial cds
* MG772933.1 Bat SARS-like coronavirus isolate bat-SL-CoVZC45, complete genome
* MN908947.3 Severe acute respiratory syndrome coronavirus 2 isolate Wuhan-Hu-1, complete genome
* MN988668.1 Severe acute respiratory syndrome coronavirus 2 isolate 2019-nCoV WHU01, complete genome
* MN988669.1 Severe acute respiratory syndrome coronavirus 2 isolate 2019-nCoV WHU02, complete genome

Analyses and Findings GC Content The GC content of each genome was calculated. The GC content is the percentage of nucleotides in a DNA sequence that are either guanine (G) or cytosine (C). The GC content was relatively similar across the genomes, with slight variations.

Amino Acid Composition The amino acid composition of each genome was calculated. The most abundant amino acids in all genomes were Leucine (L), followed by Serine (S), Threonine (T), and Valine (V). This information can provide insights into the protein-coding potential of each genome.

Sequence Similarity The sequence similarity between each pair of genomes was calculated. The genomes MN988668.1 and MN988669.1 were found to be identical, with a similarity of 100%. The other genomes had varying degrees of similarity.

Motif Finding Two motif finding tasks were performed. The first task searched for the motif 'ATG', which is the start codon in the genetic code and indicates the start of a protein-coding region. The 'ATG' motif was quite common in these genomes.

The second task searched for the motifs 'LLQY' and 'GTNGTKR' which are significant motifs of  Sars Covid 2. The 'LLQY' motif was present in some of the genomes but not all. The 'GTNGTKR' motif was present only in the genomes MN988668.1 and MN988669.1. These findings could indicate specific features or functions that are present in some of the viruses but not others.

Conclusion This project provided valuable insights into the structure and composition of the seven virus genomes. The analyses revealed similarities and differences between the genomes, which could be further explored in future studies. The findings could also be used as a basis for more detailed analyses, such as genome annotation, comparative genomics, and phylogenetic analysis.
